src/ui/public/agg_response/hierarchical/_build_split.js

```diff
@@ -1,16 +1,43 @@
+import _ from 'lodash';
 import collectKeys from 'ui/agg_response/hierarchical/_collect_keys';
 import AggResponseHierarchicalTransformAggregationProvider from 'ui/agg_response/hierarchical/_transform_aggregation';
 export default function biuldSplitProvider(Private) {
   let transformer = Private(AggResponseHierarchicalTransformAggregationProvider);
+  function showLabelCount(vis) {
```

Comment:
If we're only going to support this in point_series for now (and I'm fine with that) we should remove these changes.